### PR TITLE
Access to Gettext.plural_keys

### PR DESCRIPTION
**Pull request by nikosd**
Allowing Gettext.plural_keys access gives to someone the opportunity to actually do custom mapping of pluralization forms to gettext msgids which currently is broken for all non english-like locales.

### CODE REVIEW

File: lib/i18n/gettext.rb [hunk 1]
```diff
@@ -8,11 +8,12 @@ module Gettext
...
-      # returns an array of plural keys for the given locale so that we can
-      # convert from gettext's integer-index based style
+      # returns an array of plural keys for the given locale or the whole hash
+      # of locale mappings to plural keys so that we can convert from gettext's
+      # integer-index based style
       # TODO move this information to the pluralization module
-      def plural_keys(locale)
-        @@plural_keys[locale] || @@plural_keys[:en]
+      def plural_keys(*args)
+        args.length == 0 ? @@plural_keys : @@plural_keys[args.first] || @@plural_keys[:en]
```
Comment (radar) [inline lib/i18n/gettext.rb]: `args.empty?` would look better here imo.

**Comment by radar** [comment]
@nikosd Could you please provide a test for this as @tigrish mentioned?

**Comment by tlatsas** [comment]
@radar added a test, however all tests with Gemfile.rails* fail with:
```
Your Gemfile.lock is corrupt. The following gem is missing from the DEPENDENCIES
section: 'i18n'
```

**Comment by radar** [comment]
Thank you @tlatsas. I think the issue you're seeing will have been fixed by #355.

**Comment by tlatsas** [comment]
Should I merge master on this branch?

**Comment by radar** [comment]
@tlatsas Yes please.

**Comment by atzorvas** [comment]
@radar merged master and suite is ok, are we ok in order for you to proceed with the merge to upstream?

Thanks :+1:

**Comment by radar** [comment]
LGTM. Thanks for your work on this!

**Comment by atzorvas** [comment]
@radar maybe you should also remove the "waiting" label, cheers

**Pull request merged**